pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool]
+[tool.poetry]
+name = "guake"
+version = "4.0.0"
```

Comment:
try to use https://github.com/mtkennerly/poetry-dynamic-versioning

Reply:
I was under the impression we were working up to a major release. See your comment here:
https://github.com/Guake/guake/pull/1711#issuecomment-584732273

But no problem. Should I put the version at `3.7.0`then?

Reply:
That’s why I think deriving version from the got history is much powerful. The day I do the release I can choose whatever the version I think it is best.

So my answer would be: don’t put any version here, check with the poetry plugin if it does the job